dev-resources/src/main/resources/findbugs/findbugs-exclude.xml

```diff
@@ -4,6 +4,7 @@
 		<Or>
 			<Package name="~.*\.avro" />
 			<Package name="~.*\.protobuf" />
+			<Package name="~.*\.gwql.parse" />
```

Comment:
The dev-resources pom is not SNAPSHOT (its release) so any change to this file typically should correspond with an incremented the version of that pom